Geometry/TrackerCommonData/data/PhaseI/pixbar.xml

```diff
@@ -930,7 +1096,7 @@
    <Translation x="[zero]" y="[zero]" z="-[PixelBarrelPipe_FlangeToST_Z]" />
  </PosPart>
 
- 
+ <!-- CONNECTORS --> 
  <!--

```

Comment:
@ghugo83 - I'd suggest to remove commented out parts.

Reply:
@ianna 
As you want, I found it nice to add a few comments, I find it makes it much easier to avoid errors and know which section of the detector is being described :/